doc/api/esm.md

```diff
@@ -743,6 +748,27 @@ _isMain_ is **true** when resolving the Node.js application entry point.
 >    1. Throw an _Unsupported File Extension_ error.
 > 1. Return _legacyMainURL_.
 
+**PACKAGE_EXPORTS_RESOLVE**(_packageURL_, _packagePath_, _pjson_)
+> 1. Assert: _pjson_ is not **null**.
+> 1. Assert: _pjson.exports_ is an Object.
```

Comment:
In the real implementation, we would also reach this point for non-object values (as long as they aren't `null` or `undefined`).